community/examples/xpk-n2-filestore/xpk-n2-filestore.yaml

```diff
@@ -32,7 +32,7 @@ vars:
   slice_cpu_count: 64
   xpk_version: v0.8.0
   kueue_version: v0.11.1
-  jobset_version: v0.7.2
+  jobset_version: 0.10.1
```

Comment:
Can the versions be updated to match the jobset standard of using a 'v' at the beginning? For example: `v0.10.1`